DCK-51 Added attendee paragraph with functionality to set attende names.

andriyun opened the pull request:
## Issue

> **PLEASE, PROVIDE SHORTLY WHAT YOU HAVE MADE AT THIS PR**

## Steps for review:

- [ ] Buy ticket
- [ ] Go to My ticket tab on `/user` page
- [ ] Test all possible way to add user information.

## General checks
- [ ] All coding styles are fulfilled and there are no any issues reported by CodeSniffer.
- [ ] All tests are running and there are no failed tests reported by CI.
- [ ] Documentation has been updated according to PR changes.
- [ ] Steps for review have been provided according to PR changes.

Thank you for your сontribution and help to Drupal Ukraine!

andriyun commented on docroot/modules/custom/dckyiv_commerce/src/Controller/DckyivCommerceController.php:
```diff
@@ -0,0 +1,53 @@
+    /** @var ParagraphInterface $attendee_paragraph */
+    $attendee_paragraph = Paragraph::create(['type' => 'attendee']);
+    $attendee_paragraph->isNew();
+    $form = \Drupal::service('entity.form_builder')->getForm($attendee_paragraph, 'default');
```
Fixed

andriyun commented on docroot/modules/custom/dckyiv_commerce/src/Controller/DckyivCommerceController.php:
```diff
@@ -0,0 +1,53 @@
+      || $commerce_order_item->getOrder()->getCustomerId() != $user->id()) {
+      throw new AccessDeniedHttpException();
+    }
+    $form = \Drupal::service('entity.form_builder')->getForm($attendee_paragraph, 'edit');
```
Fixed

andriyun commented:
@ivan-berezhnov free to review this PR :)

anpolimus commented on docroot/modules/custom/dckyiv_commerce/src/Plugin/Field/FieldFormatter/AttendeeFormatter.php:
```diff
@@ -84,14 +88,18 @@ public function viewElements(FieldItemListInterface $items, $langcode) {
           'attendee' => [
             '#type' => 'link',
-            '#title' => $this->t('Add attendee info'),
+            '#title' => $this->t('Додати'),
```
English should be basic language.

anpolimus commented on docroot/modules/custom/dckyiv_commerce/dckyiv_commerce.routing.yml:
```diff
@@ -2,7 +2,7 @@ dckyiv_commerce.attendee_edit:
   defaults:
     _controller: '\Drupal\dckyiv_commerce\Controller\DckyivCommerceController::attendeeFormEdit'
-    _title: 'Edit attendee'
+    _title: 'Iнформація про учасника'
```
Basic language is English.

anpolimus commented on docroot/modules/custom/dckyiv_commerce/src/Plugin/Field/FieldFormatter/AttendeeFormatter.php:
```diff
@@ -66,15 +66,19 @@ public function viewElements(FieldItemListInterface $items, $langcode) {
           '#type' => 'link',
           '#weight' => 10,
-          '#title' => $empty ? $this->t('Add info') : $this->t('Edit'),
+          '#title' => $empty ? $this->t('Додати інфо') : $this->t('Редагувати'),
```
Basic language is English

anpolimus commented on docroot/modules/custom/dckyiv_commerce/src/Plugin/Field/FieldFormatter/AttendeeFormatter.php:
```diff
@@ -66,15 +66,19 @@ public function viewElements(FieldItemListInterface $items, $langcode) {
           '#weight' => 10,
-          '#title' => $empty ? $this->t('Add info') : $this->t('Edit'),
+          '#title' => $empty ? $this->t('Додати інфо') : $this->t('Редагувати'),
           '#url' => Url::fromRoute('dckyiv_commerce.attendee_edit', [
```
This route is deleted.

andriyun replied:
This route is ok. Here is line where it's added https://github.com/drupal-ukraine/dcamp-site/pull/82/files#diff-6d8d937740d20646932a00e67ab56bc8R1
Probably you mixed up this route with 'dckyiv_commerce.attendee_add'.
Removed mention of 'add' route

andriyun commented:
@anpolimus I've fixed changes. Please check again.

anpolimus commented:
@andriyun , thank for your last updates.
Your changes are great.
remaining things:
- [ ] Change payment/complete page based on new feature;
- [ ] Change content of receipt email

andriyun commented:
@anpolimus 
Could we move remaining things from your comment to follow up tasks and merge this PR?
There we can describe changes we missing.

anpolimus commented:
1. @andriyun Важлива річ, яку треба зробити при створенні Attendee параграфа 
- треба з даних профіля FirstName Last Name прокинути ці дані в поля параграфа, який ми створюємо
2. Додати лінк на payment/complete сторінку
3. Зробити кастомний лист

andriyun commented:
@anpolimus 
> треба з даних профіля FirstName Last Name прокинути ці дані в поля параграфа, який ми 
створюємо
Done
> Додати лінк на payment/complete сторінку
Done 
> Зробити кастомний лист
Done
+ Added styles for "my tickets" page


// @TODO
Render attendees info inside custom email.
I've implemented solution but view is empty at the current moment.
